tests/conftest.py

```diff
@@ -216,15 +192,19 @@ async def anon_client(app: FastAPI) -> AsyncGenerator[AsyncClient]:
         yield client
 
 
-@pytest.fixture
+@pytest.fixture(ids=["shared", "subdomain"], params=[False, True])
```

Comment:
Oh neat, I didn't know fixtures could be parameterized!

Reply:
Yeah, I didn't either until I was trying to figure out how to do this, and then it took me a while to figure out how to assign nice names to the options, but it's a very cool feature and very magical!